docs/additional-configs.md

```diff
@@ -53,9 +53,26 @@ namespace:
 ## Configuring CloudEvents notifications
 
 When configured so, Tekton can generate `CloudEvents` for `TaskRun`,
-`PipelineRun` and `Run`lifecycle events. The main configuration parameter is the
+`PipelineRun` and `CustomRun`lifecycle events. The main configuration parameter is the
 URL of the sink. When not set, no notification is generated.
 
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-events
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+data:
+  formats: tektonv1
+  sink: https://my-sink-url
+```
+
+The sink used to be configured in the `config-defaults` config map.
+This option is still available, but deprecated, and will be removed.
```

Comment:
could you add this to the deprecations table?

Reply:
I definitely will. I was a bit put of by the fact that I need to put things like date and release number in there, but I can start with a basic entry and update it once things are merged and released

Reply:
Done - deprecating a config parameter in favour of a new one that provides the same functionality is not really covered (or in the scope of) the API compatibility policy. Since the config option has been around for a while and there's no real extra maintenance burden in keeping it around, I marked it for removal in 9 months